lib/FluxibleContext.js

```diff
@@ -134,9 +140,10 @@ function executeActionInternal(context, action, payload, resolve, reject) {
  *  and done as parameters
  * @param {Object} payload The action payload
  * @param {Function} [done] Method to be called once action execution has completed
+ * @param {Object} meta Metadata to keep track of the current flow of executing actions
  * @return {Promise} executeActionPromise Resolved with action result or rejected with action error
  */
-FluxContext.prototype.executeAction = function executeAction(action, payload, done) {
+FluxContext.prototype.executeAction = function executeAction(action, payload, done, meta) {
```

Comment:
could we maintain BC by adding meta to the payload as a property? feels odd to not have done last.